.cruft.json

```diff
@@ -0,0 +1,16 @@
+{
+    "template": "https://github.com/HERMES-SOC/hermes_instrument.git",
+    "commit": "35cde43220942734867e5207ff69464435120c1a",
```

Comment:
Is this the right commit hash? Seems like it is behind by two commits and should be 79736e586df655ae227b232fa9f685adb7d508e3

Reply:
Good catch, though the latest is from the last PR `2374d640e38b0c3a4931b04f5d8570c9a10300a4`, so I've gone ahead and used that one.